### ISSUE TITLE
Downloading short length of a VOD will cause a audio desync

### ISSUE THREAD

**Issue by Mindreflex** [lay295/TwitchDownloader#67]
I was downloading VODs which were cut down to 1-3 minutes long, when I started watching it most of them had audio desync, I looked into it and it had first 1-2secs of video footage cut out and left with an audio only for that time.

I was downloading these VODs instead of clips because I wanted more context to the clips I had.

**Comment by lay295** [comment]
An example vod and screenshot of crop settings would be helpful

**Comment by Mindreflex** [comment]
![down](https://user-images.githubusercontent.com/70490756/91670889-5aa2e480-eb2a-11ea-9240-0dc63390af8c.png)
![desync](https://user-images.githubusercontent.com/70490756/91670893-5ecf0200-eb2a-11ea-953a-e2f63ab95106.png)

**Comment by lay295** [comment]
Yeah I can reproduce it, i'll try and look into the reason and fixes for it when I have some free time.

**Comment by lay295** [comment]
Yeah, not really sure how to fix this without re-encoding the files which I really don't wanna do... I've spent an hour on this trying to fix it without avail. Looks like with ffmpeg many people have the issue, but most of the fixes seem to have to do with re-encoding. I'm sure there are probably some arguments out there that will fix it, but I've tried many and it's not working for me. I wondered how TwitchLeecher handled the same task, and it also has the desync problem in Vegas.

So I'm making a compromise since I don't think having the A/V desync in Vegas is really acceptable. Cutting in this program will no longer be accurate, up to 10s at the beginning or end (so 20s total). However, the audio and video streams will be the same length and won't have the sync issue. 

I may try and tackle this issue again in the future to restore accurate cutting, but I'm busy with school so probably not soon.
